src/mudlet-lua/lua/geyser/GeyserColor.lua

```diff
@@ -148,6 +149,22 @@ function Geyser.Color.parse(red, green, blue, alpha)
   return r, g, b, a
 end
 
+--- Searches for a close match to 'color' in the color_table
+-- @param color the color name you're trying to find
+function Geyser.Color.find_color_name(color)
+  if type(color) ~= string then
```

Comment:
```suggestion
  if type(color) ~= "string" then
```

Reply:
how bizarre, I wonder what swallowed the quotes. Thanks for catching.